queries.sql

```diff
@@ -1,3 +1,24 @@
 /*Queries that provide answers to the questions from all projects.*/
+/*Find all animals whose name ends in "mon"*/
+SELECT * from animals WHERE name like '%mon';
 
-SELECT * from animals WHERE name = 'Luna';
+/*List the name of all animals born between 2016 and 2019*/
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+/*List the name of all animals that are neutered and have less than 3 escape attempts*/
+SELECT name from animals WHERE neutered = TRUE AND escape_attempts < 3;
+
+/*List the date of birth of all animals named either "Agumon" or "Pikachu"*/
+SELECT date_of_birth from animals WHERE name = 'Agumon' or name = 'Pikachu';
+
+/*List name and escape attempts of animals that weigh more than 10.5kg*/
+SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
+
+/*Find all animals that are neutered*/
+SELECT * from animals WHERE neutered = 'TRUE';
+
+/*Find all animals not named Gabumon*/
+SELECT * from animals WHERE name != 'Gabumon';
```

Comment:
- [ ] You can also use this 👇🏾 

```sql
SELECT * FROM animals WHERE name <> 'Gabumon';
```

In SQL, <> is a comparison operator commonly used to test for inequality. In the given query, <> is used to exclude rows where the name column value is equal to 'Gabumon'.

Reply:
@bobb-Rob  Thank you for the suggestion,I'll put it in practice👍